### PR TITLE
CoC arxivops 782

### PR DESCRIPTION
**Pull request by jimentwood**

### CODE REVIEW

File: help/policies/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Policies related to authors:
+
+- [Moderation](../moderation) 
+- [Privacy](privacy_policy)
+- [Terms of submission](../terms_of_submission)
+- [Authorship policy](../registerhelp)
+
+Policies related to moderators:
+
+- All of the above plus the moderator guidelines and policies link from moderator user accounts.
```
Comment (erickpeirson) [inline help/policies/code_of_conduct.md]:
```suggestion
- All of the above, plus the moderator guidelines and policies linked from moderator user accounts.
```

File: help/policies/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+arXiv is an open access resource. We are dedicated to the permanent custodial preservation of the scholarly record and to the continued dissemination of scholarly scientific research. We are sustained by a robust community of scientists from around the world who trust us with their work and ideas, and by the financial support of individuals and organizations. 
+
+arXiv looks to the needs of the international scientific community when we set priorities and while we engage in continual development. We are a partner in ongoing open access development and pleased to share our 30 years of experience.
+
+### Ethics and Responsibility
+
+arXiv community members should uphold intellectual honesty and ethics based on community and academic standards when writing and handling scholarly works. We strive to be objective and transparent in our decisions about policies, standards, and practices.
+
+Community members assume responsibility to represent themselves and their work honestly, to be accountable for their own decisions, and to follow arXiv [policies](#policies). Community members should respect the boundary between personal privacy and scholarly openness.
```
Comment (erickpeirson) [inline help/policies/code_of_conduct.md]: Oddly the ``policies](#policies)`` link is not getting rendered correctly (https://marxdown-coc-arxivops-782.development.arxiv.org/help/policies/code_of_conduct) -- the markdown is correct, so will treat as a bug: https://arxiv-org.atlassian.net/browse/ARXIVNG-2178

Comment (erickpeirson) [reply help/policies/code_of_conduct.md]: image

File: help/policies/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+If you have questions or concerns regarding the arXiv CoC, please contact:
+
+- [Oya Rieger](mailto:oyr1@cornell.edu), arXiv Program Director, or
+- [Steinn Sigurdsson](mailto:ss3783@cornell.edu), arXiv Scientific Director, or 
+- [Licia Verde](mailto:liciaverde@icc.ub.edu), Chair of arXiv Scientific Advisory Board, or
+- [Greg Morrisett](mailto:jgm19@cornell.edu), Dean of the Faculty of Computing and Information Science at Cornell University, or
+- [Cornell University Ombudsman](https://www.ombudsman.cornell.edu/)
+
+<span id="policies"></span>
```
Comment (mhl10) [inline help/policies/code_of_conduct.md]: Intentional?

Comment (jimentwood) [reply help/policies/code_of_conduct.md]: Yes, the markdown converter treats the span like an anchor. The headings turn into anchors in markdown, but headings are subject to change.

File: help/policies/code_of_conduct.md (hunk 2 above)
Comment (jimentwood) [reply help/policies/code_of_conduct.md]: Maybe just need to rerun the checks? I had a couple edits to get the link right.

Comment (erickpeirson) [reply help/policies/code_of_conduct.md]: Oh weird, looks good now

File: help/policies/code_of_conduct.md (hunk 3 above)
Comment (mhl10) [reply help/policies/code_of_conduct.md]: Sounds good. Forgot about that behavior.

**Pull request merged**